pages/cartPage.js

```diff
@@ -0,0 +1,32 @@
+const { expect } = require('@playwright/test');
+const { verifyItemsVisible } = require('../utils');
+const { formatProductName } = require('../helpers');
+
+exports.CartPage = class CartPage {
+  constructor(page) {
+    this.page = page;
+    this.checkoutButton = page.locator('[data-test="checkout"]');
+    this.inventoryItem = page.locator('[data-test="inventory-item"]');
+  }
+
+  async removeItemFromCart(productName) {
```

Comment:
Function returns page locator instead of removing item, so I would rename it `getRemoveItemButton()` or something similar.